BraveRewards.xcodeproj/project.pbxproj

```diff
@@ -211,6 +215,16 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
+		0A5AAECE2285E80B00F0EBC7 /* CopyFiles */ = {
+			isa = PBXCopyFilesBuildPhase;
+			buildActionMask = 2147483647;
+			dstPath = "";
+			dstSubfolderSpec = 10;
+			files = (
+				0A5AAED12285E81400F0EBC7 /* SnapKit.framework in CopyFiles */,
```

Comment:
Had to add `snapkit`, otherwise couldn't import `BraveRewardsUI` for unit tests.
Found solution here https://github.com/Quick/Quick/issues/650#issuecomment-264332214